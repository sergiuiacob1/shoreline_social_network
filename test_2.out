Distance between 1 and 2: 1Distance between 1 and 3: 2Distance between 1 and 4: 3Distance between 4 and 5: 1