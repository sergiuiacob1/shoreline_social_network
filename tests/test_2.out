Distance between 1 and 1: 0
